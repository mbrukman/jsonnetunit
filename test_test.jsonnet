local test = import "test.jsonnet";

test.suite({
    testExpectNull:   {actual: null, expect: null},
    testExpectBool:   {actual: true, expect: true},
    testExpectString: {actual: "str", expect: "str"},
    testExpectNumber: {actual: 1, expect: 1},
    testExpectArray:  {
        actual: [null, true, "str", 1],
        expect: [null, true, "str", 1],
    },
    testExpectObject:  {
        actual: {a: null, b: true, c: "str", d: 1},
        expect: {a: null, b: true, c: "str", d: 1},
    },
})
