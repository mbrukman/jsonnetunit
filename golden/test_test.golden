{
   "verify": "Passed 6 test cases"
}
